contribs/gmf/src/directives/partials/search.html

```diff
@@ -8,4 +8,5 @@
 <span class="clear-button ng-hide"
         ng-hide="!ctrl.clearButton || ctrl.input_value == ''"
         ng-click="ctrl.onClearButton()"></span>
+  <input data-ng-model="color" data-ng-change="ctrl.setStyleColor(color)"></input>
```

Comment:
Maybe related to this https://github.com/camptocamp/c2cgeoportal/wiki/Specs-%23869---Color-of-search-result